ffcx/codegeneration/ufcx.h

```diff
@@ -144,8 +145,67 @@ extern "C"
     /// element).
     ufcx_finite_element** sub_elements;
 
+    /// Pointer to data to recreate the element if it is a custom Basix element
+    ufcx_basix_custom_finite_element* custom_element;
   } ufcx_finite_element;
 
+  typedef struct ufcx_basix_custom_finite_element
+  {
+    /// Basix identifier of the cell shape
+    int cell_type;
+
+    /// The degree of the element
+    int degree;
+
+    /// Dimension of the value space for axis i
+    int value_shape_length;
+
+    /// Dimension of the value space for axis i
+    int* value_shape;
+
+    /// The number of rows in the wcoeffs matrix
+    int wcoeffs_rows;
+
+    /// The number of columnss in the wcoeffs matrix
+    int wcoeffs_cols;
+
+    /// The data contained in wcoeffs
+    double* wcoeffs;
```

Comment:
I think UFCx.h should be mostly self-contained. This doesn't really help in explaining what `wcoeffs` stands for. I understand this is for basix element, so the meaning comes from there, but few more words with possible pointer where to look at would be nice.

Reply:
I've added a short description. I'll also add a link to more info once https://github.com/FEniCS/basix/issues/404 is done